backend/cmd/api/internal/auth/middleware.go

```diff
@@ -45,7 +44,7 @@ func Middleware(next http.Handler) http.Handler {
 				return
 			}
 
-			ctx := context.WithValue(r.Context(), userCtxKey, user)
+			ctx := model.UserToContext(r.Context(), user)
```

Comment:
moved this to `package model` to avoid a circular import cycle where auth imports model imports auth